File: python/department.py
class Department:
    def __init__(self, department_id, department_name, contact_info):
        self.department_id = department_id
        self.department_name = department_name
        self.contact_info = contact_info

    def list_departments(self):
        print("Listing all departments.")

    def get_department_info(self, department_id):
        print(f"Information of department ID: {department_id}")

File: GoaOnline/src/Department.java
public class Department {
    private int departmentId;
    private String departmentName;
    private String contactInfo;

    // Constructor
    public Department(int departmentId, String departmentName, String contactInfo) {
        this.departmentId = departmentId;
        this.departmentName = departmentName;
        this.contactInfo = contactInfo;
    }

    // Methods
    public void listDepartments() {
        System.out.println("Listing all departments.");
    }

    public void getDepartmentInfo(int departmentId) {
        System.out.println("Information of department ID: " + departmentId);
    }
}
